static/src/javascripts/projects/common/modules/identity/sign-in-gate/helper.js

```diff
@@ -17,6 +17,15 @@ export const isLoggedIn = isUserLoggedIn;
 // wrapper over isInABTestSynchronous
 export const isInTest: ABTest => boolean = test => isInABTestSynchronous(test);
 
+export const getInTest: (Array<ABTest>) => ABTest = tests =>
```

Comment:
Could this be named in a slightly more descriptive way? If its just a one off for this scenario where we are running 2 tests, but they need to report as a single test, it could have a more specific name ie `getTestforMultiTestHack` ??